### PR TITLE
Revert support for issuers in details object on old API

### PR DESCRIPTION
**Pull request by jreij**
## Summary
- Support issuers in details for older api versions

### CODE REVIEW

File: issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListComponent.java [hunk 1]
```diff
@@ -35,23 +38,45 @@
...
+    private void initIssuers(@NonNull PaymentMethod paymentMethod) {
+        final List<Issuer> issuersList = paymentMethod.getIssuers();
+        if (issuersList != null) {
+            initIssuers(issuersList);
+        } else {
+            initLegacyIssuers(paymentMethod.getDetails());
+        }
+    }
+
+    private void initIssuers(@NonNull List<Issuer> issuerList) {
```
Comment (caiofaustino) [inline issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListComponent.java:57]: Maybe rename one of the 2 methods? I think it's a bit confusing to have 2 `initIssuers` even though they take different parameters. Your choice.

**Review by caiofaustino — DISMISSED** [review]

File: components-core/src/main/java/com/adyen/checkout/components/model/paymentmethods/PaymentMethod.java [hunk 2]
```diff
@@ -37,6 +37,7 @@
     private static final String FUNDING_SOURCE = "fundingSource";
     private static final String ISSUERS = "issuers";
     private static final String CONFIGURATION = "configuration";
+    private static final String DETAILS = "details";
```
Comment (caiofaustino) [inline components-core/src/main/java/com/adyen/checkout/components/model/paymentmethods/PaymentMethod.java:40]: Maybe add a comment saying this is from the old API versions?

File: issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListComponent.java (hunk 1 above)
Comment (jreij) [reply issuer-list/src/main/java/com/adyen/checkout/issuerlist/IssuerListComponent.java:57]: 👍

File: components-core/src/main/java/com/adyen/checkout/components/model/paymentmethods/PaymentMethod.java (hunk 2 above)
Comment (jreij) [reply components-core/src/main/java/com/adyen/checkout/components/model/paymentmethods/PaymentMethod.java:40]: 👍

**Review by caiofaustino — APPROVED** [review]

**Pull request merged**